### PR TITLE
Add Linters

### PR DESCRIPTION
**Pull request by jerymey**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+test.node
```
Comment (LionRouge1) [inline .gitignore:1]: - [ ] Nice work making use of `.gitignore` :clap: However the `test.md` file is still in your repo. Kindly remove it from your repo, and make sure it ignores by the git system. 
Please note that the file name is `test.md` not `test.node`

File: test.md
Comment (LionRouge1) [inline test.md:1]: - [ ] Kindly remove this file from your repo and make sure it ignores by the git system as requested in the project requirement (see image)
Screenshot from 2023-05-31 09-56-45

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @jerymey, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good commit :heavy_check_mark: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
- [ ] Nice work using the recommended Microverse [README template](https://github.com/microverseinc/readme-template) for this project. :clap: However, when adopting this template, kindly do the following:

- Remove the Microverse logo. If you have your logo, you can use it, otherwise, I would advise you to remove the logo. :+1:

- Personalize the project name and project summary to the name of your project and a summary of what you implemented. :+1:

- Remove all placeholder texts that describe what should be done in each section sunglasses

- In the _**Tech stack section of the README**_, kindly remove the technologies listed in the Client, Server and Database subsections. Those sections contain technologies that were not used in this project. Kindly remove the Server and Database subsections and replace those in the Client subsections with HTML and CSS. :+1: :sunglasses:

- In the _**Key Features section**_, kindly add 1-3 things that you accomplished in this project. I would also advise you to get used to adding these features because you would do that for every project in Microverse and possibly in the professional world. :+1:

- If you are not required to deploy the application, you can remove the _**Live Demo section of the README.md**_ file. If you are required to deploy, then the link to the deployed project should be pasted under this section. :+1:

- In the _**Getting Started**_ Section, kindly do the following for the subsections:

  - In the -**Prerequisite subsection**_, kindly add packages or applications that would be needed to start this application on the local machine. You can include NPM, a browser, a code editor, git and any other thing you can think of. +1
  - In the _**Setup subsection**_, kindly include ways to clone this project and change the directory to the directory of the project. See the example below as a guide:
  ```sh
  cd my-folder
  git clone git@github.com:myaccount/my-project.git
  cd my-project
  ```
- In the _**Install subsection**_, you can indicate ways to install the linters locally and also to run them locally. :+1: :sunglasses:
- The _**Usage subsection**_ should contain ways to run the project in the browser if while the Run tests subsections should contain the commands for running the tests written in the project. +1 sunglasses
- The _**Authors**_ section should contain the social media handles of the owner of the repository and all other collaborators
- The _**Future Features**_ section of the README.md file. The Future Features section should contain 1-3 features that can be implemented in the future. Most times, this section acts as a reference to you and future collaborators whenever you want to improve your projects in the future computer :sunglasses:
- The Show Your Support Section should communicate to people ways they can help advance the project or give a star to the project if they like it :+1: :sunglasses:
- The _**Acknowledgement**_ section should contain information, links and appreciation for those who spurred or contributed to the success of the project :computer:
- The FAQ section is the only optional section and it should contain frequently asked questions about the project and answers to those questions :sunglasses: :+1:
- The LICENSE section should contain permissions given as regards the use of the project. I would advise you to follow the guide on the right side of the article that I would paste below to set a valid LICENSE :+1:
https://choosealicense.com/licenses/mit/

https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository

- [ ] Please, add a descriptive title to your Pull Request reflecting the changes you have made to the project. You might not notice it right now, but it will help you know what's in the PR later.

- [ ] Please add a summary describing in more detail the changes introduced. This will help other developers know what to expect from your project. Kindly take a step back to review how to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

- [ ] Your branch name doesn't follow GitHub flow ( A branch should have a name of a feature or action describing what feature is implemented or changes that are made in the branch). Therefore I suggest you create a new branch and use the correct name and make sure you put the link of this PR in the new PR's description that you will create from the new branch.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jerymey** [comment]
Please can you help me?

I need help creating a new branch and follow the GitHub flow. also, how to
check that the test.md file is indeed ignored on my repository?

Le mer. 31 mai 2023 à 11:00, AVLESSI Matchoudi ***@***.***> a
écrit :

**Pull request closed without merging**